content/course-specific-material/introR-NDWSC-Nov17.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "ND WSC Intro to R Training"
+date: "2017-11-21"
+slug: "introR-NDWSC-Nov17"
+output: USGSmarkdowntemplates::hugoTraining_course
+parent: Course Specific Material
+---
+December 12-14, 2017 in Bismarck, ND
+
+### Installation
+
+See [R Installation Instructions](/installr) for information on what software should be installed prior to the course. 
+
+### Tentative schedule
+
+**Day 1**
+
+* 08:00 am - 08:30 am -- Instructors available for questions
+* 08:30 am - 09:30 am -- [Introduction](/intro-curriculum/Introduction)
+* 09:30 am - 10:00 am -- [Example workflow](/intro-curriculum/data/data_workflow.Rmd)
+* 10:00 am - 10:15 am -- *Break*
+* 10:15 am - 12:00 pm -- [Get](/intro-curriculum/Get)
+* 12:00 am - 01:00 pm -- *Lunch*
```

Comment:
I'm assuming you just copied Sacramento and I incorrectly wrote 12:00 am here :)